apps/examples/echo/rpmsg-echo.c

```diff
@@ -85,7 +85,8 @@ int app(struct rpmsg_device *rdev, void *priv)
 int main(int argc, char *argv[])
 {
 	void *platform;
-	struct rpmsg_device *rpdev;
+	struct rpmsg_device *rpdev = NULL;
+	struct rpmsg_virtio_device *rpvdev;
```

Comment:
The examples should be as agnostic as possible about how the RPmsg is sent. So i would prefer to to not introduce  rpmsg_virtio_device *rpvdev in exemples.
intead do you try to use following method to retrieve the rpmsg_virtio_device  struct in platform_release_rpmsg_vdev?
`struct rpmsg_virtio_device *rvdev = metal_container_of(rpdev, struct rpmsg_virtio_device, rdev);`